bridges/AlfaBankByBridge.php

```diff
@@ -0,0 +1,83 @@
+<?php
+class AlfaBankByBridge extends BridgeAbstract {
+
+	const MAINTAINER = 'lassana';
+	const NAME = 'AlfaBank.by Новости';
+	const URI = 'https://www.alfabank.by';
+	const DESCRIPTION = 'Уведомления Alfa-Now — новости от Альфа-Банка';
```

Comment:
IMHO the Russian text should be translated into English.

Reply:
There are Russian labels in the upstream already: https://github.com/RSS-Bridge/rss-bridge/blob/df408fb8bc362aefafc164cffbced51fd0274746/bridges/Drive2ruBridge.php#L5-L75

Reply:
While I'm also not a fan, for bridges that are very country specific, I dont mind language specific labels and selectors. If you are using a russian bank that has russian news, chances are you speak russian and can read the labels.